core/domain/exp_services.py

```diff
@@ -887,6 +1125,14 @@ def compute_exploration_contributors_summary(exploration_id):
     """Returns a dict whose keys are user_ids and whose values are
     the number of (non-revert) commits made to the given exploration
     by that user_id. This does not count commits which have since been reverted.
+
+    Args:
+        exploration_id: str.The exploration id of the exploration domain object on 
+                        which number of non-revertable commits are to be counted.
+
+    Returns:
+        A dict with user_ids as keys and number(non-revert) of commits made
```

Comment:
dict. The keys are all user_ids who have made commits to the given exploration, and the corresponding values are the numbers of commits made by each user. Commits that revert to an earlier version, or forward commits which have since been reverted, are excluded.